src/script_chainer/gui/page/script_setting_interface.py

```
@@ -259,13 +294,40 @@ def __init__(self, config: ScriptConfig, index: int = 0, parent=None,
+        if invalid_msg is not None:
+            _show_warning(self.window(), '配置不合法', invalid_msg)
+            return
+
+        script_path = self.config.script_path
+        args_list: list[str] = []
+        args_str = self.config.script_arguments or ''
+        if args_str.strip():
+            args_list = shlex.split(args_str, posix=False)
+
+        display = os.path.basename(script_path)
+        cmd = [script_path] + args_list
```

Ruff (0.15.9)

[warning] Consider `[script_path, *args_list]` instead of concatenation

Replace with `[script_path, *args_list]`

(RUF005)